README.md

```diff
@@ -202,3 +202,56 @@ make jetstream-controller
 # you use a public one.
 nats-server -DV -js
 ```
+
+Build Docker image
+```sh
+make jetstream-controller-docker jetstreamVersion=1.2.3
+```
+
+## NATS Server Config Reloader
+
+On Kubernetes v1.12+ clusters it is possible to enable on-the-fly reloading of
```

Comment:
Better to point to the Helm charts instead of the NATS Operator: https://github.com/nats-io/k8s/blob/98057cec26ca65660a0007bdd82341c188f37f5b/helm/charts/nats/values.yaml#L220-L223

Reply:
Okay, updated the doc. How's this?
https://github.com/nats-io/nack/pull/19/commits/8169eeb6d10dceee18a357a0943eb4bcdbc2cc5a?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5